api/node/package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "slint-ui",
   "version": "1.9.0",
-  "main": "index.js",
-  "types": "index.d.ts",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
```

Comment:
I think it's a good idea to place the generated files into the `dist/` sub-directory. Not in scope of this PR, but as a follow-up suggestion, it might make sense to place the remaining generated files into `dist` as well, specifically the napi generated ones.